vispy/visuals/mesh.py

```diff
@@ -328,6 +383,11 @@ def _update_data(self):
                 self._colors.set_data(np.zeros((0, 4), dtype=np.float32))
         self.shared_program.vert['position'] = self._vertices
 
+        if self.shading == 'textured_globe':
+            jpg = Image.open('Platecarretissot.png')
```

Comment:
In pull request, talk about ways around this? Can the shader be objectified to be customisable?